Neos.Neos/Classes/Fusion/Helper/BackendUserHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace Neos\Neos\Fusion\Helper;
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Eel\ProtectedContextAwareInterface;
+
+/**
+ * get the current user from the backend
+ */
+class BackendUserHelper implements ProtectedContextAwareInterface {
```

Comment:
It may be better to add a specific BackendTranslationHelper instead of exposing the backend user completely to fusion.